compiler/src/dotty/tools/dotc/ast/TreeInfo.scala

```diff
@@ -134,10 +134,10 @@ trait TreeInfo[T <: Untyped] { self: Trees.Instance[T] =>
       case _ => argss
     loop(tree, Nil)
 
-  /** All term arguments of an application in a single flattened list */
+  /** All type and term arguments of an application in a single flattened list */
```

Comment:
`allArguments` is used by `Annotation#arguments`, which is used by lots of code. How can we be sure the change does not break anything?

Reply:
The tests don't show any problem, probably because most annotations don't have type parameters. But we can keep the old behavior: 12ad08527ef1cc89326e672d89eee4a2102ebbdb.